todo_app/data/session_items_service.py

```diff
@@ -1,8 +1,8 @@
 from flask import session
```

Comment:
Is there any reason to keep this file now, or can we get rid of it?

Reply:
As it exists in the git history I think I'm comfortable removing it :)